### ISSUE TITLE
GetBuilder widget.initState?.call(this) before assign controller 

### ISSUE THREAD

**Issue by AlexSmirnov9107** [jonataslaw/getx#2354]
GetBuilder widget.initState?.call(this) before assign controller

![image](https://user-images.githubusercontent.com/32673518/168240688-3674eef6-04e0-417f-85a9-11080860a1e8.png)

https://github.com/jonataslaw/getx/blob/b5ef10a65231eec23bd5fd302dc724d1e5c62346/lib/get_state_manager/src/simple/get_state.dart#L420

**Comment by tobytraylor** [comment]
Yes this is an issue and it's not symmetrical with GetX().  Would be nice for them to have the same behavior.
